superset-frontend/src/filters/components/Select/SelectFilterPlugin.tsx

```diff
@@ -18,31 +18,76 @@
  */
 import {
   AppSection,
+  DataMask,
   ensureIsArray,
+  ExtraFormData,
   GenericDataType,
+  JsonObject,
   smartDateDetailedFormatter,
   t,
   tn,
 } from '@superset-ui/core';
-import React, { useEffect, useState } from 'react';
+import React, { useCallback, useEffect, useReducer, useState } from 'react';
 import { Select } from 'src/common/components';
-import { FIRST_VALUE, PluginFilterSelectProps, SelectValue } from './types';
+import { debounce } from 'lodash';
```

Comment:
Apparently it's better to `import debounce from 'lodash/debounce'`, as it results in smaller bundle size (and we probably should apply that in other places in superset).
References:
https://stackoverflow.com/questions/35250500/correct-way-to-import-lodash
https://www.blazemeter.com/blog/the-correct-way-to-import-lodash-libraries-a-benchmark

Reply:
Good to know 👍